airbyte-integrations/connectors/source-github/source_github/spec.json

```
@@ -9,7 +9,7 @@
     "properties": {
       "access_token": {
         "type": "string",
-        "description": "Log into Github and then generate a <a href=\"https://github.com/settings/tokens\"> personal access token</a>.",
+        "description": "Log into Github and then generate a <a href=\"https://github.com/settings/tokens\"> personal access token</a>. Separate multiple tokens with \",\"",
```

Adding some more context to why someone might want to use multiple tokens. 
```suggestion
        "description": "Log into Github and then generate a <a href=\"https://github.com/settings/tokens\"> personal access token</a>. To load balance your API quota consumption across multiple API tokens, input multiple tokens separated with \",\"",
```